charts/ingress-nginx/templates/admission-webhooks/validating-webhook.yaml

```diff
@@ -12,10 +12,10 @@ webhooks:
   - name: validate.nginx.ingress.kubernetes.io
     rules:
       - apiGroups:
-          - extensions
           - networking.k8s.io
         apiVersions:
           - v1beta1
```

Comment:
cant you remove this as well?

Reply:
No, if we want to support k8s between 1.16 and 1.19